### ISSUE TITLE
User Interface

### ISSUE THREAD

**Issue by red-led** [RandoApp/Rando-android#215]
Hi!

Now there are 3 screens in app. But first of them (with "who to use", "About" and "Log out") is not what you want to see every time. So, how about interface like this?

![rando](https://cloud.githubusercontent.com/assets/1540305/10560763/6caf50e2-751d-11e5-8890-43501272a83c.png)

Menu has margins just to show blocks, it's meant to be on top and full width (see material design apps like gmail).

**Comment by resetnow** [comment]
This might be a good proposal, but this project seems abandoned :(

**Comment by red-led** [comment]
Hm... I'm receiving some photos, so not user-abandoned. Or i'ts just old photos pool?

**Comment by resetnow** [comment]
No, people actually use it, but the last commit was about a year ago, and there are some bugs of same age. I wonder who pays for server-side things.Отправлено из мобильной Яндекс.Почты: http://m.ya.ru/ymail23:34, October 23, 2015, "red_led" <notifications@github.com>:Hm... I'm receiving some photos, so not user-abandoned. Or i'ts just old photos pool? —Reply to this email directly or view it on GitHub.

**Comment by xp-vit** [comment]
@red-led Thanks for your proposal. We are currently working on new design and your ideas are considered. This is really appreciated!

> Hm... I'm receiving some photos, so not user-abandoned. Or it's just old photos pool?

Application is fully functional and all photos you receive are real photos taken by real users recently. No any pool of photos is used. :wink:

@resetnow We are coming back from hibernation starting developing new features and fixing bugs. :rocket: Stay tuned! :radio:
